02 - Community Expansions/02.05 All Community Expansions/Themes/Illusion.md

```diff
@@ -1,16 +1,16 @@
 ---
 aliases:
-- 
-tags: 
-- 
-publish: true
+  -
+tags:
+  -
+publish: false
```

Comment:
Note to self: we don't currently un-publish old plugins and themes.

There will be a project to review this, consistently, at some point in the future.
But for now, if this is merged, it will result in broken links in the site, as other locations will include links to this page, such as the machine-generated MoC in this directory.